Sources/Receipt/ReceiptDetailTableViewCell.swift

```diff
@@ -28,6 +28,13 @@ final class ReceiptDetailTableViewCell: UITableViewCell {
         super.init(coder: aDecoder)
     }
 
+    func configure(configuration: (title: String, value: String)) {
+        textLabel?.text = configuration.title
+        textLabel?.accessibilityIdentifier = configuration.title
+        detailTextLabel?.text = configuration.value
+        detailTextLabel?.accessibilityIdentifier = "(configuration.title)_value"
```

Comment:
It has to be "\(configuration.title)_value" to get dynamically value of configuration.title else it will be same for all the labels

Reply:
fixed